From 3817cd18bbfc80ce01aa3616276dae366c0239e3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Tue, 27 Jan 2026 15:45:47 +0100
Subject: [PATCH] mswsock: add SIO_UDP_NETRESET

Signed-off-by: LIU Hao <lh_mouse@126.com>
---
 mingw-w64-headers/include/mswsock.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mingw-w64-headers/include/mswsock.h b/mingw-w64-headers/include/mswsock.h
index c953271..c1f8582 100644
--- a/mingw-w64-headers/include/mswsock.h
+++ b/mingw-w64-headers/include/mswsock.h
@@ -36,6 +36,7 @@ extern "C" {
 #define TCP_BSDURGENT 0x7000
 
 #define SIO_UDP_CONNRESET _WSAIOW(IOC_VENDOR,12)
+#define SIO_UDP_NETRESET _WSAIOW(IOC_VENDOR,15)
 #if (_WIN32_WINNT < 0x0600) && (_WIN32_WINNT >= 0x0501)
 #define SIO_SOCKET_CLOSE_NOTIFY _WSAIOW(IOC_VENDOR,13)
 #endif /* >= XP && < VISTA */
-- 
2.53.0.windows.1

